_posts/GaKay/2016-02-03-Chapter-5-Exercise.md

```diff
@@ -0,0 +1,14 @@
+---
+author: gao14g
+layout: post
+title: "Ga Kay's Chapter 5 Loops Exercises"
+---
+
+Exercise 1:
+<iframe src="https://trinket.io/embed/python/f0ff3117ba" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2:
+<iframe src="https://trinket.io/embed/python/e253dc0298" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+Overall, I didn't think these exercises were too bad. The main difficulty of this assignment was making sure my logic was correct otherwise the loop wouldn't work. One issue I did run into was with the min and max for exercise 2. At first, I intialized the variables to be equal to 0. I didn't notice the problem at first with the max but I did with the min. I had an if statement that said if the user input was less than the min, change the min to the user input. A lot of times, I didn't put in a number less than 0 so the min never changed. I used some resources online and found that I could intialize the variables with "None", and I was able to resolve my issue.
```

Comment:
Nice. [our textbook](http://ils.unc.edu/~eah13/textbook/05-iterations.html#maximum-and-minimum-loops) also has this solution.